hack/jenkins/e2e.sh

```diff
@@ -434,6 +434,18 @@ case ${JOB_NAME} in
     : ${PROJECT:="kubernetes-jenkins"}
     ;;
 
+  # Runs slow tests on GCE 1.1 cluster, sequentially.
+  kubernetes-gce-slow-release-1.1)
+    : ${E2E_CLUSTER_NAME:="gce-slow-release-1.1"}
+    : ${E2E_NETWORK:="gce-slow-1-1"}
+    : ${GINKGO_TEST_ARGS:="--ginkgo.focus=$(join_regex_no_empty \
+          ${GCE_SLOW_TESTS[@]:+${GCE_SLOW_TESTS[@]}} \
+          )"}
+    : ${KUBE_GCE_INSTANCE_PREFIX:="gce-slow-1-1"}
+    : ${PROJECT:="kubernetes-jenkins"}
```

Comment:
For future reference, because the discussion on email makes me think I'll be closing this pr out, I just do this as myself or do we use some special bot for the accounts?